docs/source/tutorial/data_examples.rst

```diff
@@ -63,6 +69,134 @@ You can also specify that a folder be renamed when it is uploaded to Citrination
 
 .. literalinclude:: /code_samples/data/upload_dir_with_dest.py
 
+Selecting a Custom Ingester
+---------------------------
+
+Finding an Ingester by ID
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``list_ingesters`` method returns all of the custom ingesters available on
+your Citrination deployment. It accepts no parameters, and returns an instance
+of :class:`IngesterList`, which itself contains instances of :class:`Ingester`.
+
+The unique field of an ``Ingester`` is its ``id``, so a particular ``Ingester``
+can be located via the ``IngesterList#find_by_id`` method.
+
+.. literalinclude:: /code_samples/data/find_ingester_by_id.py
+
+Finding an Ingester by Searching its Attributes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you don't know the ``id`` of an ingester, you can also search through an
+``IngesterList`` via the ``where`` method, which supports searching through a
+variety of attributes that can be found in the ``Ingester.SEARCH_FIELDS`` constant.
+
+.. literalinclude:: /code_samples/data/find_ingester_using_where.py
+
+.. attention::
+  Note that the `IngesterList#where` method returns a new instance of `IngesterList`,
+  and requires indexing into the `IngesterList#ingesters` attribute to ultimately
+  select an ingester.
+
+Viewing an Ingester's Arguments
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once you have an `Ingester`, you can check its optional and required arguments
+via the `Ingester#arguments` attribute.
+
+.. literalinclude:: /code_samples/data/ingester_arguments.py
+
+Uploading Data Using a Custom Ingester
+--------------------------------------
+
+The ``upload_with_ingester`` method allows for custom ingesters to be used when
+uploading a file.
+
+This method is parameterized with the following values:
+
+* **dataset_id** - The integer value of the ID of the dataset to which
+  you will be uploading
+* **source_path** - The path to the file that you want to upload and for the
+  ingester to then process
+* **ingester** - The custom ingester you want to use
```

Comment:
id? name? do either work?

Reply:
This is intended to be an instance of the Ingester class. I'll capitalize and accent the word `Ingester`